### PR TITLE
BC-70 COA COC Lab Supervisor

### PR DESCRIPTION
**Pull request by Lunga001**
## Description of the issue/feature this PR addresses
The Lab Supervisor field will be used on the COA and COC. From the contacts we could not tell who    
the supervisor is hence the we put the Lab Supervisor field on the Lab
https://jira.bikalabs.com/browse/BC-70
https://github.com/bikalabs/bika.lims/issues/

## Current behavior before PR
     Lab Information does not have Laboratory Supervisor field

## Desired behavior after PR is merged
     To set up a Lab Supervisor of the Lab
--
I confirm I have read the [Bika LIMS Developer Guidelines][1] and that I have
tested the PR thoroughly and coded it according to [PEP8][2] standards.

[1]: https://github.com/bikalabs/bika.lims/wiki/Bika-LIMS-Developer-Guidelines

[2]: https://www.python.org/dev/peps/pep-0008

### CODE REVIEW

**Comment by Lunga001** [comment]
The Lab Supervisor field will be used on the COA and COC. From the contacts we could not tell who the supervisor is hence the we put the Lab Supervisor field on the Lab

**Comment by ramonski** [comment]
Thanks @Lunga001, that's ok for me, but we should add GH issues in the future to keep a reference to the requirement in the Changelog. Please also try to put yourself into the user perspective, therefore you don't have to describe the sense/functionality to me, but to the users that are asking themselves why they have to fill in this information and where it makes a difference.

**Comment by Lunga001** [comment]
Thank you @ramonski I've put on the description

**Comment by ramonski** [comment]
Thanks @Lunga001, well done.

File: bika/lims/content/organisation.py [hunk 1]
```diff
@@ -28,6 +30,19 @@
             label=_("Name"),
         ),
     ),
+    StringField(
```
Comment (Nihadness) [inline bika/lims/content/organisation.py]: I think this should not be on the base Organisation schema.

**Comment by Nihadness** [comment]
@ramonski , @Lunga001 
do you already use that field somewhere in the reports? If yes, could you please link a PR or a commit you added it to code?
Thanks in advance!

**Comment by Lunga001** [comment]
Hi @Nihadness ,

We've used the LaboratorySupervisor to customise BIKA ACE https://github.com/Lunga001/ace.lims/blob/master/ace/lims/browser/analysisrequest/publish.py#L276
and https://github.com/Lunga001/ace.lims/blob/master/ace/lims/browser/analysisrequest/templates/reports/ACE_COA.pt#L224

**Comment by Nihadness** [comment]
Okay, thank you very much @Lunga001 !

**Pull request merged**